jkube-kit/enricher/generic/src/main/java/org/eclipse/jkube/enricher/generic/NetworkingV1IngressGenerator.java

```diff
@@ -57,7 +56,7 @@ protected static Ingress generate(KubernetesListBuilder listBuilder, ServiceBuil
         if (serviceMetadata == null) {
             log.info("No Metadata for service! ");
         }
-        if (isExposedService(serviceMetadata) && shouldCreateExternalURLForService(serviceBuilder, log)) {
+        if (shouldCreateExternalURLForService(serviceBuilder, log)) {
```

Comment:
The behavior of this might now be different, since before we were checking for Web Ports (See `ExposeEnricher.WEB_PORTS`)
Is this addressed elsewhere?